Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM python:3.8
-LABEL maintainer="Robin Cole @robmarkcole"
+FROM malenurse/ubuntu-20.04-base
+LABEL developer="Robin Cole @robmarkcole"
+LABEL docker_maintainer="Zach McDonough @malenurse"
```

Comment:
Lets just remove these 2 labels, people will just make issues on this repo anyway